docs/howto/logging.txt

```diff
@@ -0,0 +1,336 @@
+.. _logging-how-to:
+
+================================
+How to configure and use logging
+================================
+
+.. seealso::
+
+    * :ref:`Django logging reference <logging_ref>`
+    * :ref:`Django logging overview <logging-explanation>`
+
+Django provides a working :ref:`default logging configuration
+<logging-default-configuration>` that is readily extended.
+
+Make a basic logging call
+=========================
+
+To send a log message from within your code, you place a logging call into it.
+
+.. admonition:: Don't be tempted to use logging calls in ``settings.py``.
+
+    The way that Django logging is configured as part of the ``setup()``
+    function means that logging calls placed in ``settings.py`` may not work as
+    expected, because *logging will not be set up at that point*. To explore
+    logging, use a view function as suggested in the example below.
+
+First, import the Python logging library, and then obtain a logger instance
+with :py:func:`logging.getLogger`. Provide the ``getLogger()`` method with a
+name to identify it and the records it emits. A good option is to use
+``__name__`` (see :ref:`naming-loggers` below for more on this) which will
+provide the name of the current Python module as a dotted path::
+
+    import logging
+
+    logger = logging.getLogger(__name__)
+
+And then in a function, for example in a view, send a record to the logger::
```

Comment:
From a newbie's perspective, is it worth pointing out above this, that _generally speaking_ the convention is that the `logger = getLogger(...)` declaration is done at module level, so they don't question where they should be putting such things?